x-pack/legacy/plugins/maps/public/layers/sources/source.js

```diff
@@ -115,6 +115,10 @@ export class AbstractSource {
     return AbstractSource.isIndexingSource;
   }
 
+  isPushedData() {
```

Comment:
Don't think sources need to be aware of pushed data. The concept can exist at the layer level

Reply:
There are 3 comments covering this, I'll tackle it here. I don't see this as a layer property. If we think of our source classes as literally describing the qualities of their respective sources, this is where I'd expect to find it. I'd probably even ask myself, "Is this a pushed/injected data source". It's possible we might view this differently in the future as we expand our injected data options, maybe we can reevaluate it at that point?